boop-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: boop-service
+  labels:
+    app: boop-app
+spec:
+  selector:
+    app: boop-app
+  ports:
+    - name: boop-http-service
+      protocol: TCP
+      port: 80
+      targetPort: 5000
+      nodePort: 30000
```

Comment:
Why are we using nodeport?  I guess because we don't have an ingress controller?